ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -175,13 +180,11 @@ else
 		netman update -i eth0
 	fi
 
-	#
-	# If the airframe has been previously reset SYS_AUTCONFIG will have been set to 1 and other params will be reset on the next boot.
-	#
+	# To trigger a parameter reset during boot SYS_AUTCONFIG was set to 1 before
 	if param greater SYS_AUTOCONFIG 0
 	then
-		# Reset params except Airframe, RC calibration, sensor calibration, flight modes, total flight time, and next flight UUID.
-		param reset_all SYS_AUTOSTART RC* CAL_* COM_FLTMODE* LND_FLIGHT* TC_* COM_FLIGHT*
+		# Reset parameters except airframe, parameter version, RC calibration + flight modes, sensor + airspeed calibration, total flight time, flight UUID
+		param reset_all SYS_AUTOS* SYS_PA* RC* COM_FLTMODE* CAL_* ASPD_S* LND_FLIGHT* TC_* COM_FLIGHT*
```

Comment:
What's actually the argument for not resetting COM_FLTMODE?

Reply:
I would vote for resetting it. AFAIK it was purely left for convenience.

Reply:
Convenience but I'm not chaing it in this pr since it's not the goal here.